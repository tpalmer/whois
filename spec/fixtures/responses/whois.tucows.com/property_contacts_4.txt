Registrant:
 The Eye Center
 594 E. Millsap
 Fayetteville, AR 72703
 US

 Domain name: SEE2020NOW.COM


 Administrative Contact:
    Thomas, Jennifer  jthomas@see2020now.net
    594 E. Millsap
    Fayetteville, AR 72703
    US
    479-442-2020
 Technical Contact:
    Hostmaster, Contact  hostmaster@gcmcomputers.com
    101 W. Mountain St.
    Suite 101
    Fayetteville, AR 72701
    US
    479-521-3100    Fax: 479-521-3188



 Registration Service Provider:
    GCM Computers, Inc., hostmaster@gcmcomputers.com
    479-521-3100
    479-521-3188 (fax)
    http://www.gcmcomputers.com



 Registrar of Record: TUCOWS, INC.
 Record last updated on 09-Nov-2011.
 Record expires on 06-Feb-2014.
 Record created on 06-Feb-2002.

 Registrar Domain Name Help Center:
    http://tucowsdomains.com

 Domain servers in listed order:
    NS2.GCMCOMPUTERS.COM   
    NS3.GCMCOMPUTERS.COM   
    NS1.GCMCOMPUTERS.COM   


 Domain status: clientTransferProhibited
                clientUpdateProhibited

The Data in the Tucows Registrar WHOIS database is provided to you by Tucows
for information purposes only, and may be used to assist you in obtaining
information about or related to a domain name's registration record.

Tucows makes this information available "as is," and does not guarantee its
accuracy.

By submitting a WHOIS query, you agree that you will use this data only for
lawful purposes and that, under no circumstances will you use this data to:
a) allow, enable, or otherwise support the transmission by e-mail,
telephone, or facsimile of mass, unsolicited, commercial advertising or
solicitations to entities other than the data recipient's own existing
customers; or (b) enable high volume, automated, electronic processes that
send queries or data to the systems of any Registry Operator or
ICANN-Accredited registrar, except as reasonably necessary to register
domain names or modify existing registrations.

The compilation, repackaging, dissemination or other use of this Data is
expressly prohibited without the prior written consent of Tucows.

Tucows reserves the right to terminate your access to the Tucows WHOIS
database in its sole discretion, including without limitation, for excessive
querying of the WHOIS database or for failure to otherwise abide by this
policy.

Tucows reserves the right to modify these terms at any time.

By submitting this query, you agree to abide by these terms.

NOTE: THE WHOIS DATABASE IS A CONTACT DATABASE ONLY.  LACK OF A DOMAIN
RECORD DOES NOT SIGNIFY DOMAIN AVAILABILITY.


