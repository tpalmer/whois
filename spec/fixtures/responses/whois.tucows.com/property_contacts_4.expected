#admin_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_ADMIN
  should: %s[0].name         == "Thomas, Jennifer"
  should: %s[0].organization == nil
  should: %s[0].address      == "594 E. Millsap"
  should: %s[0].city         == "Fayetteville"
  should: %s[0].zip          == "72703"
  should: %s[0].state        == "AR"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "479-442-2020"
  should: %s[0].fax          == nil
  should: %s[0].email        == "jthomas@see2020now.net"

#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type         == Whois::Record::Contact::TYPE_TECHNICAL
  should: %s[0].name         == "Hostmaster, Contact"
  should: %s[0].organization == nil
  should: %s[0].address      == "101 W. Mountain St.\nSuite 101"
  should: %s[0].city         == "Fayetteville"
  should: %s[0].zip          == "72701"
  should: %s[0].state        == "AR"
  should: %s[0].country_code == "US"
  should: %s[0].phone        == "479-521-3100"
  should: %s[0].fax          == "479-521-3188"
  should: %s[0].email        == "hostmaster@gcmcomputers.com"
